### PR TITLE
fix a few typo's in README.md file

### PR DESCRIPTION
**Pull request by Jeff-Meendering**
Looked through the README file and found a few typos.  Just cleaned a few things up.

### CODE REVIEW

**Comment by drduh** [comment]
Can you remove the additional files which got included?

**Comment by Jeff-Meendering** [comment]
Alright, I've just removed the additional files... Please let me know if there are any other changes needed.

File: README.md [hunk 1]
```diff
@@ -763,15 +763,15 @@ See [Mac OSX Startup](http://www.malicious-streams.com/article/Mac_OSX_Startup.p
...
+Also, disable **Bing Web Searches** in the Spotlight preferences to avoid your search queries being sent to Microsoft.
 
 See [fix-macosx.com](https://web.archive.org/web/20180817061520/https://fix-macosx.com/) for detailed instructions.
 
 > If you've upgraded to OS X 10.10 "Yosemite" and you're using the default settings, each time you start typing in Spotlight (to open an application or search for a file on your computer), your local search terms and location are sent to Apple and third parties (including Microsoft).
 
  **Note** This Web site and instructions may no longer work on macOS Sierra - see [issue 164](https://github.com/drduh/macOS-Security-and-Privacy-Guide/issues/164).
 
-For comparison to Windows 10, see <https://fix10.isleaked.com/>
+For acomparison to Windows 10, see <https://fix10.isleaked.com/>
```
Comment (blochberger) [inline README.md:774]: Missing space.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ This guide is also available in [简体中文](https://github.com/drduh/macOS-Se
   * [OpenBSM audit](#openbsm-audit)
   * [DTrace](#dtrace)
   * [Execution](#execution)
-  * [Network](#network)
+  * [Network](#network)This
```
Comment (blochberger) [inline README.md:71]: Left "This" here on accident?

**Review by blochberger — CHANGES_REQUESTED** [review]
Spotted two minor mistakes. Looks good otherwise.

**Pull request closed without merging**